### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by sayan-rudder**
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/54)

### CODE REVIEW

File: analytics.js [hunk 1]
```diff
@@ -760,6 +766,7 @@ if (process.browser) {
   if (methodArg.length > 0 && methodArg[0] == "load") {
     let method = methodArg[0];
     methodArg.shift();
+    console.log("=====from init, calling method:: ", method)
```
Comment (sajalrudderlabs) [inline analytics.js]: Logger format need to be aligned as mentioned in earlier comments

File: analytics.js [hunk 2]
```diff
@@ -772,6 +779,7 @@ if (process.browser) {
       let event = [...instance.toBeProcessedArray[i]];
       let method = event[0];
       event.shift();
+      logger.debug("=====from init, calling method:: ", method)
```
Comment (sajalrudderlabs) [inline analytics.js:782]: Logger format need to be aligned as mentioned in earlier comments. it needs to be aligned

File: analytics.js [hunk 3]
```diff
@@ -168,22 +168,26 @@ class Analytics {
...
         object.readyCallback
       );
 
+      logger.debug("==registering ready callback===")
       object.on("ready", object.executeReadyCallback);
 
       object.clientIntegrationObjects.forEach(intg => {
+        logger.debug("===looping over each successful integration====")
         if (!intg["isReady"] || intg["isReady"]()) {
+          logger.debug("===letting know I am ready=====", intg["name"])
```
Comment (sajalrudderlabs) [inline analytics.js:190]: Logger format need to be aligned as mentioned in earlier comments

File: analytics.js [hunk 4]
```diff
@@ -168,22 +168,26 @@ class Analytics {
...
       object.executeReadyCallback = after(
         object.clientIntegrationObjects.length,
         object.readyCallback
       );
 
+      logger.debug("==registering ready callback===")
       object.on("ready", object.executeReadyCallback);
 
       object.clientIntegrationObjects.forEach(intg => {
+        logger.debug("===looping over each successful integration====")
```
Comment (sajalrudderlabs) [inline analytics.js:188]: Logger format need to be aligned as mentioned in earlier comments

File: analytics.js [hunk 5]
```diff
@@ -168,22 +168,26 @@ class Analytics {
...
       object.clientIntegrationObjects = [];
       object.clientIntegrationObjects = object.successfullyLoadedIntegration;
 
+      logger.debug("==registering after callback===", object.clientIntegrationObjects.length)
       object.executeReadyCallback = after(
         object.clientIntegrationObjects.length,
         object.readyCallback
       );
 
+      logger.debug("==registering ready callback===")
```
Comment (sajalrudderlabs) [inline analytics.js:184]: Logger format need to be aligned as mentioned in earlier comments

**Comment by sayan-rudder** [comment]
will update the logger format over the time
@sajalrudderlabs

**Pull request merged**